pyharmony/__main__.py

```diff
@@ -280,11 +280,7 @@ def ha_change_channel(token, ip, port, channel):
     client = ha_get_client(token, ip, port)
     status = client.change_channel(channel)
     client.disconnect(send_close=True)
-    if status:
-        return True
-    else:
-        logger.error('Unable to change the channel')
-        return False
+    return 0
```

Comment:
I don't like removing the logging statement, the change on client.py looks fine though.